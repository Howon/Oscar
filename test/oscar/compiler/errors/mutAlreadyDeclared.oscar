message testNow()


actor Tester() {
  mut int x = 1;
  mut int x = 2;
  
  receive = {
    | testNow() => {
        println(x);
    }
  }
}

def main() => unit = {
  actor<Tester> tester = spawn actor<Tester>();
  message<testNow>() |> tester;
}