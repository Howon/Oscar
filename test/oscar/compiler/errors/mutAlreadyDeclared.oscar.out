Mutable var x declared already